cmd/gdeploy/commands/sso/sso.go

```diff
@@ -176,12 +179,45 @@ var configureCommand = cli.Command{
 Create a group called 'Granted Administrators' in your identity provider and copy the group's ID.
 Users in this group will be able to manage Access Rules.
 `)
-		adminGroupPrompt := &survey.Input{Message: "The ID of the Granted Administrators group in your identity provider:", Default: dc.Deployment.Parameters.AdministratorGroupID}
-		err = survey.AskOne(adminGroupPrompt, &dc.Deployment.Parameters.AdministratorGroupID)
+
+		err = idp.IdentityProvider.Init(ctx)
+		if err != nil {
+			return err
+		}
+
+		err = deploy.RunConfigTest(ctx, idp.IdentityProvider)
```

Comment:
We should only run the test if `update == false`
The UDE = I have updated my config and I see a test passed prompt, then straight away I see tests passed again.

Alternatively we can actually just omit the test here all together because I think it is unessecary.

```suggestion
		
```